public/app/features/dashboard/dashboardCtrl.js

```diff
@@ -57,6 +57,7 @@ function (angular, $, config, moment) {
 
         dashboardKeybindings.shortcuts($scope);
 
+        $scope.evaluateTotalStats();
```

Comment:
this should not be done on dashboard load, can your remove the stats from this PR completley?